chrome/browser/ui/browser.cc

```diff
@@ -563,6 +563,25 @@ Browser::Browser(const CreateParams& params)
   }
 
   BrowserList::AddBrowser(this);
+  // recordreplay
+  content::WebContents::CreateParams recordreplay_params(
+    profile_,
+    SiteInstance::CreateForURL(
+      profile_,
+      GURL(base::StringPiece(chrome::kChromeUIRecordReplayPageURL))
+    )
+  );
+  recordreplay_params.is_never_visible = true;
+
+  recordreplay_contents_ = content::WebContents::Create(recordreplay_params);
+  recordreplay_contents_->SetDelegate(this);
+  recordreplay_contents_->GetController().LoadURL(
+    GURL(base::StringPiece(chrome::kChromeUIRecordReplayPageURL)),
+    content::Referrer(),
+    ui::PAGE_TRANSITION_TYPED, // wrong.  the user didn't type this
+    std::string()
+  );
```

Comment:
for lack of a better spot, this is where I created the hidden `WebContents` and cause it to load our chrome:// url.  afaict, we can put it anywhere we have a browser (to set the delegate) and profile, so likely we could host this code closer to where the button is created.

Reply:
Moved this to the toolbar button